dts/arm/st/l4/stm32l496Xe.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 Centaur Analytics, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+#include <st/l4/stm32l496.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(320)>;
```

Comment:
SRAM size is actually 256KB.
We need to take into account that 64Kb RAM are located into SRAM2 at a non contiguous address.

Reply:
@erwango Done.